README.md

```diff
@@ -24,7 +24,11 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
 ```ruby
-# code here
+def offer_rose(person)
+  puts "Would you take this rose, #{person}, in exchange for giving an old beggar woman shelter from the bitter cold?"
+end
+
+puts offer_rose("young prince")
```

Comment:
:+1: although you don't need to `puts` the method call since you already `puts` something in your method defintion.